packages/frontend/src/hooks/chat/useMessage.ts

```diff
@@ -139,6 +140,9 @@ export default function useMessage() {
         ...message,
       })
 
+      // Notify about the sent message (listeners can filter by message type if needed)
+      notifyWebxdcMessageSent(accountId, chatId, message)
```

Comment:
This doesn't really make sense semantically, because the message is not necessarily a message with a webxdc attachment.